--- a/src/btop.cpp
+++ b/src/btop.cpp
@@ -800,13 +800,6 @@ int main(int argc, char **argv) {
 				Logger::warning("Failed to set macos locale, continuing anyway.");
 			}
 		}
-	#else
-		if (found.empty() and Global::utf_force)
-			Logger::warning("No UTF-8 locale detected! Forcing start with --utf-force argument.");
-		else if (found.empty()) {
-			Global::exit_error_msg = "No UTF-8 locale detected!\nUse --utf-force argument to force start if you're sure your terminal can handle it.";
-			clean_quit(1);
-		}
 	#endif
 		else if (not set_failure)
 			Logger::debug("Setting LC_ALL=" + found);
